.github/workflows/ci.yml

```diff
@@ -455,40 +456,39 @@ jobs:
           chown -R opam "${GITHUB_OUTPUT}"
           sudo -u opam -E ./.github/scripts/build-apk.sh ${{ needs.build_details.outputs.branch }} ${{ matrix.os }} ${{ matrix.platform }} ${{ matrix.alpine-arch }} "${{ needs.build_details.outputs.is_rolling_release }}" "${{ needs.build_details.outputs.is_release }}" "${{ needs.build_details.outputs.minimal_exclude_deps }}"
       - name: Upload alpine packages artifacts
-        if: needs.build_details.outputs.is_fork != 'true' && matrix.os == 'alpine'
-        uses: savonet/aws-s3-docker-action@master
+        if: contains(matrix.os, 'alpine')
```

Comment:
Last I checked this action did not work either on alpine or arm64 or both..

Reply:
I want to try the hack described in the issue https://github.com/actions/runner/issues/801#issuecomment-2394425757

Reply:
I have more research to do, so not in this PR.